lib/sql.js

```diff
@@ -593,6 +593,27 @@ SQLConnector.prototype.updateAttributes = function(model, id, data, options, cb)
 };
 
 /**
+ * Replace attributes for a given model instance
+ * @param {String} model The model name
+ * @param {*} id The id value
+ * @param {Object} data The model data instance containing all properties to
+ * be replaced
+ * @param {Object} options Options object
+ * @param {Function} cb The callback function
+ * @private
+ */
+SQLConnector.prototype.replaceAttributes = function(model, id, data, options, cb) {
+  if (!isIdValuePresent(id, cb)) {
+    return;
+  }
+  var idName = this.idName(model);
+  delete data[idName];
+  var where = {};
+  where[idName] = id;
+  this.replaceAll(model, where, data, options, cb);
```

Comment:
Could you please call `this.replace` here, as that's the method defined here in code, so that the reader does not have to look up the alias replaceAll->replace?